src/main/java/org/tikv/common/util/ConcreteBackOffer.java

```diff
@@ -37,8 +39,11 @@
 public class ConcreteBackOffer implements BackOffer {
   private static final Logger logger = LoggerFactory.getLogger(ConcreteBackOffer.class);
   private final int maxSleep;
-  private final Map<BackOffFunction.BackOffFuncType, BackOffFunction> backOffFunctionMap;
-  private final List<Exception> errors;
+
+  @VisibleForTesting
+  public final Map<BackOffFunction.BackOffFuncType, BackOffFunction> backOffFunctionMap;
+
+  @VisibleForTesting public final List<Exception> errors;
```

Comment:
```suggestion
  @VisibleForTesting
  public final List<Exception> errors;
```

Reply:
It is `./dev/javafmt` format the code this way.